tox.ini

```diff
@@ -8,7 +8,7 @@ deps =
     pytest-cov==4.1.*
     coverage==7.3.*
     parameterized==0.9.*
-    windows-curses; platform_system=="Windows" and platform_python_implementation=="CPython" and python_version<"3.13"
+    windows-curses; platform_system=="Windows" and platform_python_implementation=="CPython" and python_version<"3.14"
```

Comment:
Doesn't yet support Python 3.14 (https://github.com/zephyrproject-rtos/windows-curses/issues/76)

Reply:
do you think it will be added within a reasonable time span? (<= 4-5 weeks)